desk/app/diary.hoon

```diff
@@ -202,11 +202,11 @@
     (toggle-post toggle)
   ::
       %leave-old-channels
-    =/  groups-path  /(scot %p our.bowl)/groups/(scot %da now.bowl)/groups/noun
-    =/  groups  .^(groups:g %gx groups-path)
+    =/  groups-path  /(scot %p our.bowl)/groups/(scot %da now.bowl)/groups/groups
+    =/  groups  .^(groups:v2:g %gx groups-path)
```

Comment:
`/noun` at the end _should_ still work, right? Arguably that's more robust against mark renaming shenanigans, making it preferable, but all of that _should_ be static in practice so it _shouldn't_ matter either way.

Reply:
It would work, but so far I have followed a convention of using the mark whenever it is available. (There are untyped endpoints, and for those `noun` is the proper choice). 

I can see one other reason to prefer `noun`: there is less typing when upgrading endpoints, since then the only change is bumping the version number in the path and type.